sqlalchemy-stubs/orm/scoping.pyi

```diff
@@ -1,13 +1,15 @@
 from typing import Any
 from typing import Optional
 
+from ..orm import Session as Session
```

Comment:
I think it's best to avoid `as Session` since it signals mypy that it's a public export

Reply:
right, fixed